### DISCUSSION TITLE
Add ability to track coins on the Polygon/Matic network.

### DISCUSSION THREAD

**Discussion by Drbass87** [LedgerHQ/ledger-live-desktop#3992] [Coin/Token request]
The Polygon network has gotten extremely popular recently, especially after Ethermine started allowing miners to mine Eth directly to the Polygon network.  I am requesting the ability to add the Polygon network to my Ledger Live app so I can track the value of my crypto.

While I understand that I can do this by attaching my MetaMask wallet to my Ledger account, I believe this to be a hacky solution.  As a newer user I was thrown off by how your app was telling me I could import all my ERC-20 Tokens but they didn't show up because they were on the Polygon network.  It caused much wailing and gnashing of teeth.

**Comment by dkfi** [comment]
I see that this has been added to the latest Ledger Live Desktop update. Nice!

**Comment by saiaman** [comment]
Its done but looks not syncing all transactions but just updates balances in my case....
